sysutils/macos-trash/Portfile

```diff
@@ -19,9 +19,9 @@ description         command-line program that moves items to the trash
 long_description    ${name} is a small command-line program that moves \
                     files or folders to the trash, written in Swift
 
-checksums           rmd160  0fc3fbf74364226cf9e8b90bc8fbc1edb509260e \
-                    sha256  c4472b5c8024806720779bc867da1958fe871fbd93d200af8a2cc4ad1941be28 \
-                    size    2986
+checksums           rmd160  5a73588b188ed8834ae98d5cf5702df76df7231e \
+                    sha256  95eeea2a96e5d989145da4697206062798b9f708101dc426ae5a489969619114 \
+                    size    3123
 
 minimum_xcodeversions {20 13}
```

Comment:
update the `minimum_xcodeversions` statement and remove the `if {${os.platform} eq "darwin" && ${os.major} < 20} {` block and instead set the `platforms` field appropriately.

Reply:
it looks like on the macos-14 runner it has installed:
```
  DEBUG: Xcode 15.4, CLT 16.2.0.0.1.1733547573
```

so yeah, that error message is correct as Xcode 16.1 is required and 15.4 is installed...